### PR TITLE
Accept `bowerFunnels` as `EmberBuild` option

### PR DESCRIPTION
**Pull request by seanpdoyle**
Include `bower_components` in test tree

Pass in bower dependencies so that Ember-Data's tests have access.

### CODE REVIEW

File: lib/config/test-config-tree.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = function getTestConfigTree(opts) {
   var options = opts || {};
 
   var testConfig = new Funnel(options.libPath || 'tests', {
-    include: [ /html$/, /js$/ ],
+    include: [ /html$/, /js$/, "bower_components/*" ],
```
Comment (twokul) [inline lib/config/test-config-tree.js]: `"bower_components/*"` I believe this should be a regex

Comment (seanpdoyle) [inline lib/config/test-config-tree.js]: woops, this snuck its way in there. It isn't necessary

**Comment by rwjblue** [comment]
I would prefer to simply allow "otherTrees" to be passed through (instead of changing how `getBowerTres` works).

So if in Ember Data you specified:

```
var emberTree = new Funnel('bower_components/ember', {
  destDir: 'ember'  
});

new EmberBuild({
  additionalTrees: emberTree
});
```

**Comment by seanpdoyle** [comment]
@rwjblue :ok_hand: 

is a `Funnel` instance an acceptable tree?

**Comment by rwjblue** [comment]
Yep, it is a tree.

**Comment by stefanpenner** [comment]
this seems to have gone stale. @seanpdoyle I'm also working to get your ED patch in, so I may circle back and add this feature if needed.

**Pull request closed without merging**